### PR TITLE
fix: issue in upload funxtion route and modified the test to work with the new logic

### PR DESCRIPTION
**Pull request by DestinedCodes**
O
​
## Description
I fixed some bugs caused by typo in the upload chunk function and I also modified the test  to run well with the new logic

​
## Related Issue (Link to linear ticket)
N/A

​
## Motivation and Context
I tried running the tests that existed and it gave issues since the logic alhas changed so I decided to fix the tests then I found those bugs

​
## How Has This Been Tested?
I tests by running the tests and also via the swagger document and ithe API clients

​
## Screenshots (if appropriate - Postman, etc):
​
Screenshot_20231008-115941
Screenshot_20231008-115654
Screenshot_20231008-115528
Screenshot_20231008-115318
Screenshot_20231008-114734
Screenshot_20231008-115415 
## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
​
## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [x] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

**Comment by adelajaOlamilekan** [comment]
Good

File: tests/test_blob.py [hunk 1]
```diff
@@ -3,35 +3,54 @@
...
+START_URL = "http://127.0.0.1:8000/srce/api/start-recording/"
+UPLOAD_URL = "http://127.0.0.1:8000/srce/api/upload-recording/"
 BLOB_SIZE = 1 * 1024 * 1024  # 1MB by default. Adjust as needed.
-USERNAME = "cofucan"
+USER_ID = "cofucan"
 FILENAME = "videoA"  # This should be unique for each video.
 
 
-def send_blob(blob, blob_id, is_last):
+def start_record(user_id):
```
Comment (zxenonx) [inline tests/test_blob.py:14]: Could you please add type hint and docstring there

File: tests/test_blob.py [hunk 2]
```diff
@@ -3,35 +3,54 @@
...
+        response_data = response.json()
+        video_id = response_data.get("video_id")
+        if video_id:
+            return video_id
+        else:
+            print("Video ID not found in the response.")
+    else:
+        print(f"Request failed with status code: {response.status_code}")
+
+def send_blob(video_id, blob, blob_index, is_last):
```
Comment (zxenonx) [inline tests/test_blob.py:31]: type hints and docstring here too.

**Review by zxenonx — COMMENTED** [review]
Thanks @DestinedCodes.
I left some comments

**Comment by DestinedCodes** [comment]
@zxenonx requested changes has been made

**Review by zxenonx — APPROVED** [review]
Great @DestinedCodes

**Pull request merged**